lms/validation/authentication/_lti.py

```diff
@@ -1,11 +1,9 @@
 import marshmallow
 
 from lms.models import CLAIM_PREFIX, LTIUser
-from lms.services import (
-    ApplicationInstanceNotFound,
-    LTILaunchVerificationError,
-    LTIUserService,
-)
+from lms.services.application_instance import ApplicationInstanceNotFound
+from lms.services.launch_verifier import LTILaunchVerificationError
+from lms.services.lti_user import LTIUserService
```

Comment:
Replacing these imports from the `lms.services` root with submodule imports works around existing circular dependencies between `lms.services` and `lms.validation`. See https://hypothes-is.slack.com/archives/C1MA4E9B9/p1709813420775679.

Reply:
Yeah this is a tricky one to have completely satisfying solution.

Does `from lms import services` and then using `service.XXXX` work?

It's also a workaround but avoids having to type all those modules names.

Reply:
> Does from lms import services and then using service.XXXX work?

I'd expect that to have the same issue, since both `import lms.services` and `from lms import services` will execute `lms/services/__init__.py`.